qlog/src/events/quic.rs

```diff
@@ -384,30 +384,43 @@ pub enum QuicFrameTypeName {
 // ensure it goes out on the wire. This means that deserialization of frames
 // also works automatically.
 pub enum QuicFrame {
-    Padding,
+    Padding {
+        length: Option<u32>,
+        payload_length: u32,
```

Comment:
```suggestion
        payload_length: Option<u32>,
```
Maybe? (Haven't looked at the I-D, this is just based on the other additions of `payload_length`.

Reply:
Yeah I did the same but it really is not optional in https://quicwg.org/qlog/draft-ietf-quic-qlog-quic-events.html#section-8.13.1